recipes/chemper/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "chemper" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mobleylab/chemper/archive/1.0.0.tar.gz
+  sha256: d8e1b0f29d005be6ba617f2dcb484706e4d06db1df1f6f4a26aa826e20472c7b
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
This can be noarch: python

Reply:
You don't need to deselect Windows for noarch because they are multiplatform. Staged Recipes can't re-render the repo, but once you get the feedstock the `noarch` keyword will trigger a rebuild where only Linux is used to build the package. You can remove `skip: true` even if Windows fails here - it won't matter in the feedstock!

Reply:
Thanks!